lib/galaxy/managers/citations.py

```diff
@@ -47,6 +47,7 @@ def _raw_get_bibtex(self, doi):
         doi_url = "https://doi.org/" + doi
         headers = {'Accept': 'text/bibliography; style=bibtex, application/x-bibtex'}
```

Comment:
```suggestion
        headers = {'Accept': 'application/x-bibtex'}
```
seems to work out of the box with `req.text`. Only difference is that the output is formatted with newlines and tabs (not sure that matters).

Reply:
Looks like it is just fine this way, works out of the box. I got properly rendered umlauts on galaxy page. ```apparent_encoding``` solution kind of looks better to me. If in some unrealistic case,  they decide to change default style, we are prepared :satisfied: . But I don't really mind, I can force push the stuff with headers, so which one should I use?

Reply:
I'd probably do both. For reference, this is documented at https://citation.crosscite.org/docs.html